src/Microsoft.Data.DataView/IDataView.cs

```diff
@@ -132,16 +132,18 @@ public abstract class DataViewRow : IDisposable
         public abstract ValueGetter<DataViewRowId> GetIdGetter();
 
         /// <summary>
-        /// Returns whether the given column is active in this row.
+        /// Returns whether the give column is active in this row.
         /// </summary>
-        public abstract bool IsColumnActive(int col);
+        public abstract bool IsColumnActive(DataViewSchema.Column column);
 
         /// <summary>
-        /// Returns a value getter delegate to fetch the given column value from the row.
+        /// Returns a value getter delegate to fetch the value of the given <paramref name="column"/>, from the row.
         /// This throws if the column is not active in this row, or if the type
         /// <typeparamref name="TValue"/> differs from this column's type.
         /// </summary>
-        public abstract ValueGetter<TValue> GetGetter<TValue>(int col);
+        /// <typeparam name="TValue"> is the output column's content type.</typeparam>
```

Comment:
(nit) Is it confusing to say `output column` here and below?  To the consumer of an IDataView, they are just `column`s.